independent-publisher-connectors/Brønnøysundregistrene/apiProperties.json

```diff
@@ -0,0 +1,10 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#0B576F",
+    "capabilities": [],
+    "publisher": "Ahmad Najjar",
+    "website": "https://www.brreg.no/",
```

Comment:
@ahmadn82 The property called "website" is not allowed in apiProperties.json file, please remove it.

Reply:
@sriyen-msft I've removed the property called "website"